[2024-06-30T08:31:31.284+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-30T08:31:31.308+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: northwindDWH.data_delivery manual__2024-06-30T08:31:27.348915+00:00 [queued]>
[2024-06-30T08:31:31.315+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: northwindDWH.data_delivery manual__2024-06-30T08:31:27.348915+00:00 [queued]>
[2024-06-30T08:31:31.315+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-30T08:31:31.328+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): data_delivery> on 2024-06-30 08:31:27.348915+00:00
[2024-06-30T08:31:31.336+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=117) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-30T08:31:31.337+0000] {standard_task_runner.py:63} INFO - Started process 119 to run task
[2024-06-30T08:31:31.338+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'northwindDWH', 'data_delivery', 'manual__2024-06-30T08:31:27.348915+00:00', '--job-id', '82', '--raw', '--subdir', 'DAGS_FOLDER/northwindDWH.py', '--cfg-path', '/tmp/tmp06dk42b_']
[2024-06-30T08:31:31.339+0000] {standard_task_runner.py:91} INFO - Job 82: Subtask data_delivery
[2024-06-30T08:31:31.350+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-06-30T08:31:31.379+0000] {task_command.py:426} INFO - Running <TaskInstance: northwindDWH.data_delivery manual__2024-06-30T08:31:27.348915+00:00 [running]> on host 3abb3aef8f21
[2024-06-30T08:31:31.466+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Mahmoud' AIRFLOW_CTX_DAG_ID='northwindDWH' AIRFLOW_CTX_TASK_ID='data_delivery' AIRFLOW_CTX_EXECUTION_DATE='2024-06-30T08:31:27.348915+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-30T08:31:27.348915+00:00'
[2024-06-30T08:31:31.468+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-30T08:31:31.484+0000] {northwindDWH.py:49} INFO - Data delivery started
[2024-06-30T08:31:31.484+0000] {Load_data.py:85} INFO - loading dimEmployee to db ...
[2024-06-30T08:31:31.484+0000] {Transform_data.py:48} INFO - start Extracting dim employee
[2024-06-30T08:31:31.488+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-30T08:31:31.491+0000] {Transform_data.py:57} INFO - Extract dim employees done
[2024-06-30T08:31:31.492+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-30T08:31:31.496+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-30T08:31:31.499+0000] {Load_data.py:103} INFO - SCD type 2 on dimemployee started
[2024-06-30T08:31:31.506+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-30T08:31:31.506+0000] {Load_data.py:158} INFO - loading dimproduct to db ...
[2024-06-30T08:31:31.506+0000] {Transform_data.py:25} INFO - start Extracting dim product
[2024-06-30T08:31:31.509+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-30T08:31:31.512+0000] {Transform_data.py:39} INFO - Extract dim products done
[2024-06-30T08:31:31.512+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-30T08:31:31.515+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-30T08:31:31.519+0000] {Load_data.py:172} INFO - SCD type 1 on dimproduct started
[2024-06-30T08:31:31.534+0000] {Tables_creations.py:27} INFO - Loading data into table dim_products...
[2024-06-30T08:31:31.537+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-30T08:31:31.544+0000] {Load_data.py:203} INFO - Error while loading dimProducts: integer out of range

[2024-06-30T08:31:31.544+0000] {Load_data.py:12} INFO - loading dimCustomers to db ...
[2024-06-30T08:31:31.544+0000] {Transform_data.py:7} INFO - start Extracting dim customers
[2024-06-30T08:31:31.547+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-30T08:31:31.549+0000] {Transform_data.py:16} INFO - Extract dim customer done
[2024-06-30T08:31:31.550+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-30T08:31:31.553+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-30T08:31:31.555+0000] {Load_data.py:30} INFO - SCD type 2 on dimcustomer started
[2024-06-30T08:31:31.569+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-30T08:31:31.569+0000] {Load_data.py:207} INFO - Loading factsales to db ...
[2024-06-30T08:31:31.570+0000] {Transform_data.py:67} INFO - start Extracting fact sales
[2024-06-30T08:31:31.572+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-30T08:31:31.575+0000] {Transform_data.py:78} INFO - Extract fact sales done
[2024-06-30T08:31:31.576+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-30T08:31:31.578+0000] {Tables_creations.py:27} INFO - Loading data into table fact_sales...
[2024-06-30T08:31:31.581+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-30T08:31:31.598+0000] {Tables_creations.py:41} INFO - Data loaded successfully into table fact_sales With 58 Row
[2024-06-30T08:31:31.599+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-30T08:31:31.599+0000] {Load_data.py:215} INFO - Fact_Sales loaded successfully
[2024-06-30T08:31:31.600+0000] {northwindDWH.py:54} INFO - Data delivery done
[2024-06-30T08:31:31.600+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-30T08:31:31.600+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-30T08:31:31.608+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=northwindDWH, task_id=data_delivery, run_id=manual__2024-06-30T08:31:27.348915+00:00, execution_date=20240630T083127, start_date=20240630T083131, end_date=20240630T083131
[2024-06-30T08:31:31.631+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-30T08:31:31.647+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-30T08:31:31.649+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
